### DISCUSSION TITLE
Demo of draggable elements, i.e., to change order of items?

### DISCUSSION THREAD

**Discussion by joewiz** [Jinntec/Fore#202] [Q&A]
Has anyone implemented a draggable control for changing the order of items in a list?

For example, in the todo list or kanban board demos, users might expect a handle for dragging the order/position of items within or even across lists.

**Comment by joewiz** [comment]
Here's a TEI use case for this from @wsalesky: https://markmail.org/message/roia3xvgop6h7kgv.

**Reply by JoernT** [reply]
i had a first shot at this one day. But it's not trivial to do and to be honest not high on the list currently.
